catalogs/catalog-fileset/build.gradle.kts

```diff
@@ -13,9 +13,12 @@ plugins {
 dependencies {
   implementation(project(":api"))
   implementation(project(":core"))
+  implementation(project(":common"))
```

Comment:
This directory is named for `catalog-fileset`.
What should its name be f we have another file set catalog?

Reply:
How about catalog-hadoop-fileset?

Reply:
prefer `catalog-fileset-hadoop`. Because our iceberg catalog is named to `catalog-lakehouse-iceberg`.